conf/rockstor-replication.service

```
@@ -0,0 +1,19 @@
+[Unit]
+Description=Rockstor Replication Daemon
+After=network.target rockstor-pre.service rockstor.service
+Requires=rockstor-pre.service rockstor.service
```

```suggestion
Description=Rockstor Replication Service
After=rockstor.service
Requires=rockstor.service
```

Same as for collector service re title and use of service and narrow After= and Requires=  to most immediate relevant service - that of the SGI